frontend/app/src/components/landingpages/blog/Governance.svelte

```diff
@@ -0,0 +1,122 @@
+<script lang="ts">
+    import ExternalLink from "../ExternalLink.svelte";
+    import Link from "../Link.svelte";
+    import BlogScreenshot from "./BlogScreenshot.svelte";
+</script>
+
+<h1>OpenChat Governance</h1>
+<section>
+    <h2>Introduction</h2>
+    <p>
+        Now that OpenChat is controlled by an SNS you might be wondering how to take part in the governance of OpenChat and earn voting rewards. In this blog post we'll summarise the key concepts and also give some step by step guides.    
+    </p>
+    <p>
+        A good place to start to understand more about SNS DAOs is the article <ExternalLink href="https://internetcomputer.org/sns/faq">What you need to know about SNS DAOs</ExternalLink>.
+    </p>
+    <p>
+        After that for information specific to OpenChat please read the governance section of the <Link path="/whitepaper?section=3-4">OpenChat SNS Whitepaper</Link>.
+    </p>
+</section>
+<section>
+    <h2>Neurons</h2>
+    <p>
+        CHAT neurons are very similar in their function to ICP neurons. They allow users to “time-lock” their CHAT tokens to gain voting power, which can be used to vote on proposals submitted to the SNS. Neurons can be made to follow each other in various ways such that they vote automatically, representing a form of <ExternalLink href="https://en.wikipedia.org/wiki/Liquid_democracy">liquid democracy</ExternalLink>.
+    </p>
+    <p>
+        Users can also <em>dissolve</em> their neurons to release the CHAT tokens inside the neuron.
+    </p>
+</section>
+<section>
+    <h2>Proposals</h2>
+    <p>
+        Changes to the OpenChat SNS, which include the SNS treasury of ICP and CHAT tokens, the various tokenomic parameters, and the OpenChat dapp itself, can only be made by the OpenChat SNS governance canister upon successful execution of <em>proposals</em>.
+    </p>
+    <p>        
+        Other than "Motion" proposals all other types result in some automatic action by the SNS such as...
+    </p>        
+    <ul>
+        <li>Transfer ICP or CHAT out of the treasury</li>
+        <li>Upgrade an OpenChat canister</li>
+        <li>Change SNS meta data such as the "proposal rejection cost"</li>
+        <li>Call a custom SNS function for example to add a "platform moderator"</li>
+    </ul>
+    <p>
+        Join <Link path="/nsbx4-4iaaa-aaaar-afusa-cai">this group</Link> to view, discuss, and vote on OpenChat SNS proposals.
+    </p>
+    <p>
+        Anyone who owns a neuron can use it to make proposals although this currently requires knowledge of the command line tools <ExternalLink href="https://github.com/dfinity/sdk">dfx</ExternalLink> and <ExternalLink href="https://github.com/dfinity/sns-quill#submit-a-proposal">quill</ExternalLink>. In time this process should become easier. If a proposal is rejected then 10 CHAT tokens is subtracted from the proposing neuron and burned to discourage spam.
```

Comment:
The link is to sns quill but the text just says 'quill'. Is that just because the docs being linked to are better than what's in quill?

Reply:
Ah good catch. I copied this from the SNS FAQ but it is probably out of date itself.
https://internetcomputer.org/sns/faq#how-do-i-create-a-proposal-for-an-sns-dao